ipn/prefs.go

```diff
@@ -144,6 +144,10 @@ type Prefs struct {
 	// Tailscale, if at all.
 	NetfilterMode preftype.NetfilterMode
 
+	// OperatorUser is the local machine user name who is allowed to
+	// operate tailscaled without being root or using sudo.
+	OperatorUser string `json:",omitempty"`
```

Comment:
Make []string? Would be nice to be able to grant tailscale use to multiple operators, maybe? Unsure.

Reply:
We could keep it simple now and support comma-delimited in the future if we want to add that later?

Reply:
Also, once we get to wanting to support multiple operators, that starts to sound like a group to me.

So I don't want to make this overly complicated for now. Let's address the common case (single user, single admin) and deal with the Enterprise Management Solutions(tm) later once needs are understood.